packages/medusa-js/src/resources/returns.ts

```diff
@@ -1,15 +1,16 @@
 import BaseResource from './base';
-import * as Types from '../types';
+import { AxiosPromise } from 'axios'
```

Comment:
nit: Missing linting

Reply:
can we try to enforce this through a linting rule? Hoping we could use something like [this](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md). What do you think?

Reply:
One of the other PRs include a fix, that will ensure correct linting of the client :) It is currently a `prettier` file in the client folder, that tampers with out `eslint`.